docs/live-migration.md

```diff
@@ -0,0 +1,235 @@
+# Live Migration
+
+## Introduction
+
+The Live Migration feature allows [kubevirt](kubevirt.io) virtual machines to be [live migrated](https://kubevirt.io/user-guide/operations/live_migration/)
+while keeping the established TCP connections alive, and preserving the VM IP configuration.
+These two requirements provide seamless live-migration of a KubeVirt VM using OVN-Kubernetes
+cluster default network.
+
+## Requirements
+
+- KubeVirt >= v1.0.0
+- DHCP aware guest image (fedora family is well tested, https://quay.io/organization/containerdisks)
+
+# Limitations
+
+- Only KubeVirt VMs with bridge binding pod network are supported
+- Single stack IPv6 is not supported
+- DualSack do not configure routes for IPv6 over DHCP/autoconf
+- SRIOV is not supported
+
+## Example: live migrating a fedora guest image
+
+Install KubeVirt following the guide [here](https://kubevirt.io/user-guide/operations/installation/)
+
+Create a fedora virtual machine with the annotations `kubevirt.io/allow-pod-bridge-network-live-migration` and `k8s.ovn.org/skip-ip-configuration-on-cni`:
```

Comment:
didn't you remove the `skip-ip-configuration` annotation ?

Reply:
Done